tests/secc/states/test_iso15118_2_states.py

```diff
@@ -383,3 +386,35 @@ async def test_service_discovery_req_unexpected_state(self):
             service_discovery.message.body.service_discovery_res.response_code
             is ResponseCode.FAILED_SEQUENCE_ERROR
         )
+
+    async def test_charging_status_evse_status(self):
+        charging_status = ChargingStatus(self.comm_session)
+        self.comm_session.writer = Mock()
+        self.comm_session.writer.get_extra_info = Mock()
+        self.comm_session.selected_schedule = 1
+        await charging_status.process_message(message=get_dummy_charging_status_req())
+
+        charging_status_res = charging_status.message.body.charging_status_res
+        assert charging_status_res.ac_evse_status == ACEVSEStatus(
+            notification_max_delay=0,
+            evse_notification=EVSENotification.NONE,
+            rcd=False,
+        )
+
+    async def test_charging_status_evse_status_altered(self):
+        charging_status = ChargingStatus(self.comm_session)
+        self.comm_session.writer = Mock()
+        self.comm_session.writer.get_extra_info = Mock()
+        self.comm_session.selected_schedule = 1
+
+        async def get_ac_evse_status_patch():
+            return ACEVSEStatus(
+                notification_max_delay=0,
+                evse_notification=EVSENotification.NONE,
+                rcd=True,
```

Comment:
Ok - I see you changed the rcd value for this response to be different from the simulator...